docs/client/wn-oasis.md

```diff
@@ -0,0 +1,126 @@
+```
+
+Validating the Worker Node Client
+---------------------------------
+
+To verify functionality of the worker node client, you will need to submit a test job against your CE and verify the job's output.
+
+1.  Submit a job that executes the `env` command (e.g. Run [condor\_ce\_trace](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/TroubleshootingHTCondorCE#condor_ce_trace) with the `-d` flag from your HTCondor CE)
+2.  Verify that the value of `OSG_GRID` is set to the directory of your WN Client installation
+
+### Manually Using the Worker Node Client From OASIS
+
+If you must log onto a worker node and use the Worker Node Client software directly during your login session, consult the following table for the command to set up your environment:
+
+| Worker Node OS | Run the following command…                                                                           |
+|:---------------|:-----------------------------------------------------------------------------------------------------|
+| EL 6 (32-bit)  | `source /cvmfs/oasis.opensciencegrid.org/osg-software/osg-wn-client/3.3/current/el6-i386/setup.sh`   |
+| EL 6 (64-bit)  | `source /cvmfs/oasis.opensciencegrid.org/osg-software/osg-wn-client/3.3/current/el6-x86_64/setup.sh` |
+| EL 7 (64-bit)  | `source /cvmfs/oasis.opensciencegrid.org/osg-software/osg-wn-client/3.3/current/el7-x86_64/setup.sh` |
+
+Troubleshooting
+---------------
+
+Some possible issues that may come up:
+
+- A missing softlink to the CA certs directory. To check this, run:
+
+        :::console
+        [user@client ~] $ ls -l /cvmfs/oasis.opensciencegrid.org/mis/osg-wn-client/3.3/current/el6-x86_64/etc/grid-security/
+    
+    and check that `certificates` is linked to somewhere. The fix is to yum update the `oasis-config` package to version 4 or higher. A known workaround is to run:
+
+        :::console
+        [user@client ~] $ export X509_CERT_DIR=/cvmfs/oasis.opensciencegrid.org/mis/certificates
+    
+    before any commands.
+
+How to get Help?
+----------------
+
+To get assistance please use this [Help Procedure](../common/help.md).
+
+Reference
+---------
+
+Please see the documentation on using [yum and RPM](../release/yum-basics.md), [the best practices](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/InstallBestPractices) for using yum to install software, and using [yum repositories](../common/yum.md).
+
+### Worker node contents
+
+The worker node may be updated from time to time. As of OSG 3.3.21 in February 2017, the OSG worker node client contains:
+
+-   OSG Certificates
+-   curl
+-   Fetch CRL
+-   FTS client
+-   gfal2
+-   globus-url-copy (GridFTP client)
+-   globus-xio-udt-driver
+-   ldapsearch
+-   MyProxy
+-   osg-system-profiler
+-   osg-version
+-   UberFTP
+-   vo-client (includes /etc/vomses file)
+-   VOMS client
+-   wget
+-   xrdcp
+
+To see the currently installed version of the worker node package, run the following command:
```

Comment:
This won't work with the OASIS install, right? I think we should just get rid of this entire section